docs/style.md

```diff
@@ -46,9 +46,9 @@ chmod 755 .git/hooks/pre-commit
 Note: these hooks depend on some python tools being installed in your environment. These
 can be installed using:
 ```bash
-pip install -r test-requirements.txt
+pip install .[developer]
 ```
-For more information about how to contribute to armory, see [Contributing to Armory](./contributing.md).
+For more information about how to contribute to armory, see [Contributing to Armory](/.github/CONTRIBUTING.md).
```

Comment:
Will this link correctly in `readthedocs`?

Reply:
reverted.